src/main/webapp/resources/js/pages/projects/ncbi/create/CreateNcbiSampleDetails.tsx

```diff
@@ -264,11 +285,27 @@ export default function CreateNcbiSampleDetails({
             name={["samples", sample.name, "pairs"]}
             label={i18n("CreateNcbiExport.pairs")}
             valuePropName="checked"
+            rules={[
+              ({ getFieldValue }) => ({
+                validator(_, value) {
```

Comment:
Since this validator is being used in multiple places please refactor it into a function which will accept a string (pairs, singles, etc)